[2024-06-02T20:49:33.068+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgre_exercises.prediction_result manual__2024-06-02T20:49:26.694980+00:00 [queued]>
[2024-06-02T20:49:33.071+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgre_exercises.prediction_result manual__2024-06-02T20:49:26.694980+00:00 [queued]>
[2024-06-02T20:49:33.071+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-06-02T20:49:33.080+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): prediction_result> on 2024-06-02 20:49:26.694980+00:00
[2024-06-02T20:49:33.083+0000] {standard_task_runner.py:60} INFO - Started process 2308 to run task
[2024-06-02T20:49:33.086+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'postgre_exercises', 'prediction_result', 'manual__2024-06-02T20:49:26.694980+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/create_table.py', '--cfg-path', '/tmp/tmpxv02mcnu']
[2024-06-02T20:49:33.087+0000] {standard_task_runner.py:88} INFO - Job 46: Subtask prediction_result
[2024-06-02T20:49:33.120+0000] {task_command.py:423} INFO - Running <TaskInstance: postgre_exercises.prediction_result manual__2024-06-02T20:49:26.694980+00:00 [running]> on host 35a5c0dbc0eb
[2024-06-02T20:49:33.164+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='msaipulrx@gmail.com' AIRFLOW_CTX_DAG_OWNER='saipul' AIRFLOW_CTX_DAG_ID='postgre_exercises' AIRFLOW_CTX_TASK_ID='prediction_result' AIRFLOW_CTX_EXECUTION_DATE='2024-06-02T20:49:26.694980+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-02T20:49:26.694980+00:00'
[2024-06-02T20:49:33.435+0000] {logging_mixin.py:188} INFO - Table Input
[2024-06-02T20:49:33.454+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/sklearn/base.py:443: UserWarning: X has feature names, but MLPClassifier was fitted without feature names
  warnings.warn(

[2024-06-02T20:49:33.460+0000] {logging_mixin.py:188} INFO - table output
[2024-06-02T20:49:33.465+0000] {logging_mixin.py:188} INFO -       Id  Predicted_Species             executed_timed
0      1                  0 2024-06-02 20:49:32.580306
1      2                  0 2024-06-02 20:49:32.580306
2      3                  0 2024-06-02 20:49:32.580306
3      4                  0 2024-06-02 20:49:32.580306
4      5                  0 2024-06-02 20:49:32.580306
..   ...                ...                        ...
145  146                  2 2024-06-02 20:49:32.580306
146  147                  2 2024-06-02 20:49:32.580306
147  148                  2 2024-06-02 20:49:32.580306
148  149                  2 2024-06-02 20:49:32.580306
149  150                  2 2024-06-02 20:49:32.580306

[150 rows x 3 columns]
[2024-06-02T20:49:33.489+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-06-02T20:49:33.496+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=postgre_exercises, task_id=prediction_result, execution_date=20240602T204926, start_date=20240602T204933, end_date=20240602T204933
[2024-06-02T20:49:33.514+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-06-02T20:49:33.525+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
